src/main/java/com/idrsolutions/microservice/BaseServlet.java

```diff
@@ -396,6 +396,12 @@ private boolean handleFileFromUrl(final Individual individual, final HttpServlet
             doError(request, response, "No url given", 400);
             return false;
         }
+
+        if (!(url.toLowerCase().startsWith("http://") || (url.toLowerCase().startsWith("https://"))) {
```

Comment:
```suggestion
        if (!url.toLowerCase().startsWith("http://") && !url.toLowerCase().startsWith("https://")) {
```

This is quite minor (and potentially subjective), but my suggestion would be to avoid the use of groups in conditionals unless they are required to evaluate correctly.

Including them forces the reader to scan ahead to find the closing bracket and keep a mental note of that when reading the conditions, instead of simply reading the statement left-to-right. So it's just a bit easier to read without them, imo.

There's no difference in performance as it will short-circuit either way.